shared/haml/cookie_banner.haml

```diff
@@ -0,0 +1,26 @@
+%link{rel:'stylesheet', type:'text/css', href:'/shared/css/cookie-banner.css'}
+%script{src: minifiable_asset_path("js/cookieBanner.js")}
+
+-# dashboard is passed as a parameter when the dashboard is rendering this banner.
+- dashboard ||= false
+- if dashboard
+  - message = I18n.t('cookie_banner.message')
+  - more_information = I18n.t('cookie_banner.more_information')
+  - accept = I18n.t('cookie_banner.accept')
+- else
+  - message = hoc_s(:cookie_banner_message)
+  - more_information = hoc_s(:cookie_banner_more_information)
+  - accept = hoc_s(:cookie_banner_accept)
+
+#cookie-banner{style: "display: none"}
+  .text
+    = message
+    %a{href: codeorg_url('/privacy'), target: '_blank'}= more_information
+
+  .buttons
+    %button#accept-cookies= accept
+
+:javascript
+  $(document).ready(function() {
+    window.setupCookieBanner("#{rack_env}");
+  });
```

Comment:
You mentioned not depending on jQuery in your PR description, but you're using it for the document ready handler here?

Reply:
Ah, mystery solved.  The code.org homepage defers loading of jquery (https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/index.haml#L10 and https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/jquery.haml#L3-L5) which means that `$(document).ready` is available (https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/jquery_shim.js.erb#L12) but literally nothing else in jQuery is.

Reply:
I'll add a note about this.